src/atlas/interpolation/method/fe/FiniteElement.cc

```diff
@@ -132,20 +132,26 @@ struct Stencil {
 void FiniteElement::print( std::ostream& out ) const {
     functionspace::NodeColumns src( source_ );
     functionspace::NodeColumns tgt( target_ );
+    if ( mpi::rank() == 0 ) {
+        out << "atlas::interpolation::method::FiniteElement" << std::endl;
+        out << "maxFractionElemsToTry_: " << maxFractionElemsToTry_;
+        out << " treat_failure_as_missing_value_: " << treat_failure_as_missing_value_ << std::endl; 
+    }
     if ( not tgt ) {
-        ATLAS_NOTIMPLEMENTED;
+        return;
```

Comment:
Is this something you encountered?
OK to return, but perhaps output that "tgt" is an incompatible function space or so.

Reply:
yes, the tests actually work by interpolating NodeColumns to PointCloud, which means if I run the print function as part of the test I hit this exception.

 My thinking was the rest of the function only makes sense for printing NodeColumns, however this is not the only compatible target function space.
 
  In fact as far as I can tell there is no test for the NodeColumns to NodeColumns interpolation as it stands. I think it would be good to add one, but I couldn't think of nice simple test points off the top of my head! I will have another think about this if you like!

Reply:
Thanks for elaborating. I'm happy for you to leave it as is, without any further warning or exception.